shell/components/Resource/Detail/Metadata/index.vue

```diff
@@ -30,7 +30,10 @@ const showBothEmpty = computed(() => labels.length === 0 && annotations.length =
 </script>
 
 <template>
-  <SpacedRow class="metadata ppb-3">
+  <SpacedRow
+    class="metadata ppb-3"
+    :class="$attrs.class"
```

Comment:
I noticed that a class wasn't being passed down to the `SpacedRow` even though `inherritAttrs` wasn't dissaabled.

Reply:
Turns out it's because I added two children to the template which turns of inherritAttrs